src/components/PatientBrowser.vue

```diff
@@ -93,9 +89,9 @@
                 <div class="my-2 series-list">
                   <groupable-item
                     v-for="series in getSeries(study.StudyInstanceUID)"
-                    :key="series.SeriesInstanceUID"
+                    :key="series.ITKGDCMSeriesUID"
```

Comment:
Where does this key name come from? I don't see why it refers to ITK or GDCM, it should be agnostic to that. I was thinking more of `VolumeUID` to be generic? Just my 2c.

Reply:
This was from before when the application indexed on SeriesInstanceUID rather than the generated GDCM series UID. But since we are now making our own IDs, this should be replaced with references to volumes.

Reply:
Looks like `ITKGDCMSeriesUID` is still used around in that PR?